### PR TITLE
docs: added users to PoweredBy

### PR DESCRIPTION
**Pull request by juzhiyuan**
### What this PR does / why we need it:

added users to PoweredBy

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Review by Yiyiyimu — APPROVED** [review]

**Comment by Yiyiyimu** [comment]
do we need to update the icon wall at the same time

**Comment by juzhiyuan** [comment]
> do we need to update the icon wall at the same time

Not in this PR if needed.

File: doc/powered-by.md [hunk 1]
```diff
@@ -53,6 +53,21 @@ Users are encouraged to add themselves to this page, [issue](https://github.com/
 1. <a href="https://www.xin.com/" rel="nofollow">Xin 优信二手车</a>
 1. <a href="https://www.zuzuche.com/" rel="nofollow">zuzuche 租租车</a>
 1. <a href="https://www.zybang.com/" rel="nofollow">zybang 作业帮</a>
+1. <a href="https://www.daocloud.io/" rel="nofollow">DaoCloud</a>
```
Comment (spacewander) [inline doc/powered-by.md:56]: Should be in alphabetical order

**Review by spacewander — CHANGES_REQUESTED** [review]

File: doc/powered-by.md [hunk 2]
```diff
@@ -25,49 +25,49 @@ or providing commercial products including APISIX.
 Users are encouraged to add themselves to this page, [issue](https://github.com/apache/apisix/issues/487) and PR are welcomed.
 
 1. <a href="https://www.360.cn/" rel="nofollow">360 奇虎</a>
+1. <a href="http://51tiangou.com/" rel="nofollow">51tiangou 大商天狗</a>
+1. <a href="http://aimiaobi.com/" rel="nofollow">aimiaobi 妙笔 AI</a>
 1. <a href="https://www.chaolian360.com/" rel="nofollow">chaolian 超链云商</a>
+1. <a href="https://www.daocloud.io/" rel="nofollow">DaoCloud</a>
+1. <a href="http://dian.so/" rel="nofollow">小电科技</a>
+1. <a href="https://www.didachuxing.com/" rel="nofollow">嘀嗒出行</a>
 1. <a href="http://www.cunw.com.cn/" rel="nofollow">cunw 湖南新云网</a>
```
Comment (spacewander) [inline doc/powered-by.md:34]: Disorder?

File: doc/powered-by.md [hunk 3]
```diff
@@ -25,49 +25,49 @@ or providing commercial products including APISIX.
...
 1. <a href="https://www.dasouche.com/" rel="nofollow">dasouche 大搜车</a>
 1. <a href="https://www.dataoke.com/" rel="nofollow">dataoke 大淘客</a>
 1. <a href="https://www.efactory-project.eu/" rel="nofollow">eFactory</a>
 1. <a href="https://www.ehomepay.com.cn/" rel="nofollow">ehomepay 理房通</a>
+1. <a href="https://ezone.work/" rel="nofollow">eZone 简单一点科技</a>
 1. <a href="https://fansup.mobi/" rel="nofollow">fansup</a>
 1. <a href="http://haieruplus.com/" rel="nofollow">haieruplus 海尔优家</a>
+1. <a href="https://hy.10086.cn/" rel="nofollow">中移杭研</a>
+1. <a href="https://www.htzhiyun.cn/" rel="nofollow">航天网信</a>
```
Comment (spacewander) [inline doc/powered-by.md:43]: Ditto

File: doc/powered-by.md [hunk 4]
```diff
@@ -25,49 +25,49 @@ or providing commercial products including APISIX.
...
 1. <a href="https://www.meizu.com/" rel="nofollow">Meizu 魅族</a>
+1. <a href="https://www.mingyuanyun.com/" rel="nofollow">明源云客</a>
 1. <a href="https://www.jpl.nasa.gov" rel="nofollow">NASA JPL 美国国家航空航天局 喷气推进实验室</a>
 1. <a href="http://www.163.com" rel="nofollow">Netease 网易</a>
 1. <a href="http://sinog2c.com" rel="nofollow">sinog2c 湖南国科云通</a>
 1. <a href="https://www.sinovatech.com" rel="nofollow">sinovatech 炎黄新星</a>
 1. <a href="http://taikang.com/" rel="nofollow">Taikanglife 泰康云</a>
 1. <a href="http://www.tangdou.com/" rel="nofollow">tangdou 糖豆网</a>
 1. <a href="https://cloud.tencent.com/" rel="nofollow">Tencent Cloud 腾讯云</a>
+1. <a href="https://open.youtu.qq.com/" rel="nofollow">Youtu 腾讯优图</a>
```
Comment (spacewander) [inline doc/powered-by.md:61]: This website should be treat as "Youtu"?

File: doc/powered-by.md [hunk 5]
```diff
@@ -25,34 +25,49 @@ or providing commercial products including APISIX.
...
 
 1. <a href="https://www.360.cn/" rel="nofollow">360 奇虎</a>
+1. <a href="http://51tiangou.com/" rel="nofollow">51tiangou 大商天狗</a>
+1. <a href="http://aimiaobi.com/" rel="nofollow">aimiaobi 妙笔 AI</a>
 1. <a href="https://www.chaolian360.com/" rel="nofollow">chaolian 超链云商</a>
 1. <a href="http://www.cunw.com.cn/" rel="nofollow">cunw 湖南新云网</a>
+1. <a href="https://www.daocloud.io/" rel="nofollow">DaoCloud</a>
+1. <a href="http://dian.so/" rel="nofollow">小电科技</a>
+1. <a href="https://www.didachuxing.com/" rel="nofollow">嘀嗒出行</a>
 1. <a href="https://www.dasouche.com/" rel="nofollow">dasouche 大搜车</a>
```
Comment (spacewander) [inline doc/powered-by.md:35]: Disorder?

File: doc/powered-by.md [hunk 6]
```diff
@@ -25,34 +25,49 @@ or providing commercial products including APISIX.
...
 1. <a href="http://www.tangdou.com/" rel="nofollow">tangdou 糖豆网</a>
 1. <a href="https://cloud.tencent.com/" rel="nofollow">Tencent Cloud 腾讯云</a>
+1. <a href="https://www.teamones.cn/" rel="nofollow">万思</a>
 1. <a href="http://travelsky.com" rel="nofollow"> Travelsky 中国航信</a>
 1. <a href="https://vbill.cn/" rel="nofollow">vbill 随行付</a>
 1. <a href="https://www.willclass.com/" rel="nofollow">willclass 会课</a>
 1. <a href="https://www.xin.com/" rel="nofollow">Xin 优信二手车</a>
+1. <a href="https://xueqiu.com/" rel="nofollow">雪球</a>
+1. <a href="https://open.youtu.qq.com/" rel="nofollow">Youtu 腾讯优图</a>
 1. <a href="https://www.zuzuche.com/" rel="nofollow">zuzuche 租租车</a>
```
Comment (spacewander) [inline doc/powered-by.md:68]: Disorder?

Comment (juzhiyuan) [reply doc/powered-by.md:68]: 😂

**Comment by membphis** [comment]
@juzhiyuan please merge the master branch

@idbeta you can take a look the ci too: https://github.com/apache/apisix/pull/3573/checks?check_run_id=1869908569#step:10:703

**Review by moonming — APPROVED** [review]

**Comment by membphis** [comment]
image

**Comment by juzhiyuan** [comment]
https://github.com/apache/apisix/blob/1abda84d0b9bff24ed8f74f9c2ec7d9b9f99163a/doc/powered-by.md

cc @membphis to take a look at the latest commit.

**Review by membphis — APPROVED** [review]

**Comment by membphis** [comment]
@Yiyiyimu please take a look at the output of Chaos Test: https://github.com/apache/apisix/pull/3573/checks?check_run_id=1944989987#step:9:117

**Review by tokers — APPROVED** [review]

File: doc/powered-by.md [hunk 7]
```diff
@@ -24,50 +24,50 @@ or providing commercial products including APISIX.
...
+2. <a href="http://www.163.com" rel="nofollow">Netease 网易</a>
+3. <a href="https://www.360.cn/" rel="nofollow">360 奇虎</a>
+4. <a href="http://51tiangou.com/" rel="nofollow">51tiangou 大商天狗</a>
+5. <a href="http://aimiaobi.com/" rel="nofollow">aimiaobi 妙笔 AI</a>
+6. <a href="https://www.chaolian360.com/" rel="nofollow">chaolian 超链云商</a>
+7. <a href="http://www.cunw.com.cn/" rel="nofollow">cunw 湖南新云网</a>
+8. <a href="https://www.daocloud.io/" rel="nofollow">DaoCloud</a>
+9. <a href="http://dian.so/" rel="nofollow">小电科技</a>
+10. <a href="https://www.didachuxing.com/" rel="nofollow">嘀嗒出行</a>
+11. <a href="https://www.dasouche.com/" rel="nofollow">dasouche 大搜车</a>
```
Comment (spacewander) [inline doc/powered-by.md]: Disorder

File: doc/powered-by.md [hunk 8]
```diff
@@ -24,50 +24,50 @@ or providing commercial products including APISIX.
...
+21. <a href="http://huawei.com/" rel="nofollow">Huawei 华为</a>
+22. <a href="https://www.huya.com/" rel="nofollow">虎牙</a>
+23. <a href="https://www.ihomefnt.com/" rel="nofollow">ihomefnt 艾佳生活</a>
+24. <a href="https://www.jiandanxinli.com/" rel="nofollow">jiandanxinli 简单心理</a>
+25. <a href="https://www.ke.com/" rel="nofollow">ke.com 贝壳找房</a>
+26. 2. 2. 1. <a href="https://jr.ly.com/" rel="nofollow">jr.ly 同程金服</a>
+27. <a href="https://www.meizu.com/" rel="nofollow">Meizu 魅族</a>
+28. <a href="https://www.mingyuanyun.com/" rel="nofollow">明源云客</a>
+29. <a href="https://www.jpl.nasa.gov" rel="nofollow">NASA JPL 美国国家航空航天局 喷气推进实验室</a>
+30. <a href="https://open.youtu.qq.com/" rel="nofollow">Youtu 腾讯优图</a>
```
Comment (spacewander) [inline doc/powered-by.md]: Should be ordered with `youtu`?

File: doc/powered-by.md [hunk 9]
```diff
@@ -24,50 +24,50 @@ or providing commercial products including APISIX.
...
+17. <a href="http://haieruplus.com/" rel="nofollow">haieruplus 海尔优家</a>
+18. <a href="http://www.hellowin.cn/" rel="nofollow">hellowin 好洛维</a>
+19. <a href="https://www.hellotalk.com/" rel="nofollow">HelloTalk, Inc.</a>
+20. <a href="https://www.htzhiyun.cn/" rel="nofollow">航天网信</a>
+21. <a href="http://huawei.com/" rel="nofollow">Huawei 华为</a>
+22. <a href="https://www.huya.com/" rel="nofollow">虎牙</a>
+23. <a href="https://www.ihomefnt.com/" rel="nofollow">ihomefnt 艾佳生活</a>
+24. <a href="https://www.jiandanxinli.com/" rel="nofollow">jiandanxinli 简单心理</a>
+25. <a href="https://www.ke.com/" rel="nofollow">ke.com 贝壳找房</a>
+26. 2. 2. 1. <a href="https://jr.ly.com/" rel="nofollow">jr.ly 同程金服</a>
```
Comment (spacewander) [inline doc/powered-by.md]: Why use `2. 2. 1.`?

File: doc/powered-by.md (hunk 7 above)
Comment (juzhiyuan) [reply doc/powered-by.md]: image

image

What happened 😳

File: doc/powered-by.md [hunk 10]
```diff
@@ -24,33 +24,48 @@ or providing commercial products including APISIX.
...
+1. <a href="https://www.ihomefnt.com/" rel="nofollow">ihomefnt 艾佳生活</a>
 1. <a href="https://www.jiandanxinli.com/" rel="nofollow">jiandanxinli 简单心理</a>
-1. <a href="https://jr.ly.com/" rel="nofollow">jr.ly 同程金服</a>
 1. <a href="https://www.ke.com/" rel="nofollow">ke.com 贝壳找房</a>
+1. <a href="https://jr.ly.com/" rel="nofollow">jr.ly 同程金服</a>
 1. <a href="https://www.meizu.com/" rel="nofollow">Meizu 魅族</a>
+1. <a href="https://www.mingyuanyun.com/" rel="nofollow">明源云客</a>
 1. <a href="https://www.jpl.nasa.gov" rel="nofollow">NASA JPL 美国国家航空航天局 喷气推进实验室</a>
-1. <a href="http://www.163.com" rel="nofollow">Netease 网易</a>
+1. <a href="https://open.youtu.qq.com/" rel="nofollow">Youtu 腾讯优图</a>
```
Comment (spacewander) [inline doc/powered-by.md:56]: Should be ordered with `Youtu`?

Comment (juzhiyuan) [reply doc/powered-by.md:56]: I would prefer using QQ 🤔

Comment (Yiyiyimu) [reply doc/powered-by.md:56]: > I would prefer using QQ 🤔

Maybe `Tencent` could be better than `QQ` 🤣

Comment (juzhiyuan) [reply doc/powered-by.md:56]: wow, links in this PR are ordered according to the domain 😊

Comment (Yiyiyimu) [reply doc/powered-by.md:56]: Got it 😬

**Comment by idbeta** [comment]
> @juzhiyuan please merge the master branch
> 
> @idbeta you can take a look the ci too: https://github.com/apache/apisix/pull/3573/checks?check_run_id=1869908569#step:10:703



> @juzhiyuan please merge the master branch
> 
> @idbeta you can take a look the ci too: https://github.com/apache/apisix/pull/3573/checks?check_run_id=1869908569#step:10:703

sorry, I did not find the reason for the failure. It seems that the case is unstable. @syzh and @spacewander Can you check it?
```
t/node/invalid-route.t                   (Wstat: 0 Tests: 18 Failed: 3)
  Failed tests:  16-18
  Parse errors: No plan found in TAP output
```

**Review by spacewander — APPROVED** [review]

**Comment by spacewander** [comment]
@idbeta 
Should be solved by https://github.com/apache/apisix/pull/3609.

**Comment by idbeta** [comment]
> @idbeta
> Should be solved by #3609.

thanks.

**Pull request merged**